eras/alonzo/impl/cardano-ledger-alonzo.cabal

```diff
@@ -48,6 +48,7 @@ library
 
   hs-source-dirs: src
   other-modules:
+    Cardano.Ledger.Alonzo.CertState
```

Comment:
I think you forgot to add the module to git :slightly_smiling_face: 

Happens to me all the time :smile: